app/assets/stylesheets/common/_markus.scss

```diff
@@ -1366,3 +1365,35 @@ canvas {
   min-width: 250px;
   width: 50%;
 }
+
+.exam-crop-img-container {
+  margin-right: 3px;
+  max-width: 80%;
```

Comment:
Since the parent container has `display: flex` you can be a bit more specific with these widths. The button container can have `width: 40px` to match the width you set for the buttons, and then the image container can have `flex: 1` to make it fill the space